src/MaptilerGeolocateControl.ts

```diff
@@ -166,4 +166,34 @@ export class MaptilerGeolocateControl extends GeolocateControl {
       this._updateCircleRadius();
     }
   };
+
+  _setErrorState() {
```

Comment:
Lets add a comment why we are overriding it 🙏 Maybe also a link to: https://github.com/maplibre/maplibre-gl-js/issues/2294

Reply:
Good point. I'll add that